plasma_framework/contracts/src/framework/utils/ExitPriority.sol

```diff
@@ -6,32 +6,42 @@ library ExitPriority {
 
     using PosLib for PosLib.Position;
 
+    uint256 constant private SIZEOF_TIMESTAMP = 32;
+    uint256 constant private SIZEOF_TXPOS = 54;
+    uint256 constant private SIZEOF_EXITID = 168;
+
     /**
      * @dev Returns an exit priority for a given UTXO position and a unique ID.
      * The priority for Plasma M(ore)VP protocol is a combination of 'exitableAt' and 'txPos'.
-     * Since 'exitableAt' only provides granularity of block, add 'txPos' to provide priority for a transaction.
+     * Since 'exitableAt' only provides granularity of block, adding 'txPos' gives higher priority to transactions added earlier in the same block.
      * @notice Detailed explanation on field lengths can be found at https://github.com/omisego/plasma-contracts/pull/303#discussion_r328850572
      * @param exitId Unique exit identifier
      * @return An exit priority
      *   Anatomy of returned value, most significant bits first
-     *   42 bits  - timestamp in seconds (exitable_at); we can represent dates until year 141431
-     *   54 bits  - blocknum * 10^5 + txindex; 54 bits represent all transactions for 85 years. Be aware that child chain block number jumps with the interval of CHILD_BLOCK_INTERVAL, which would be 1000 in production.
-     *   160 bits - exit id
+     *   32 bits  - The exitable_at timestamp (in seconds); can represent dates until the year 2106
+     *   54 bits  - The transaction position. Be aware that child chain block number jumps with the interval of CHILD_BLOCK_INTERVAL (usually 1000).
+                    blocknum * (BLOCK_OFFSET / TX_OFFSET) + txindex; 54 bits can represent all transactions for 85 years.
+     *   168 bits - The exit id
      */
-    function computePriority(uint64 exitableAt, PosLib.Position memory txPos, uint160 exitId)
+    function computePriority(uint64 exitableAt, PosLib.Position memory txPos, uint168 exitId)
         internal
         pure
         returns (uint256)
     {
-        return (uint256(exitableAt) << 214) | (txPos.getTxPositionForExitPriority() << 160) | uint256(exitId);
+        return (uint256(exitableAt) << (256 - SIZEOF_TIMESTAMP)) | (txPos.getTxPositionForExitPriority() << SIZEOF_EXITID) | uint256(exitId);
     }
 
     function parseExitableAt(uint256 priority) internal pure returns (uint64) {
```

Comment:
Wouldn't it be useful for a client to be able to call the parsing functions?

Reply:
In ExitPriorityWrapper

Reply:
meanwhile, actually I am slightly on the fence whether we should have this function at all or not as no production code is using this at this moment. If we only want it for testing, we can put this logic into `ExitPriorityWrapper` as alternative.

but I am not sure which is better actually.

Reply:
I am ok with that as well.

Reply:
`parseExitableAt()` is used in `ExitGameController`
 
https://github.com/omgnetwork/plasma-contracts/blob/30741a715e6acd9c95f2cc3cf4a8f77666f00950/plasma_framework/contracts/src/framework/ExitGameController.sol#L168

The 2 new parse functions aren't used, but I put them in here for consistency.

They are all available to clients in the `ExitPriorityWrapper`.

Not sure what the issue is - do you want to move just `parseExitId` and `parseTxPos` to `ExitPriorityWrapper`?

Reply:
> do you want to move just parseExitId and parseTxPos to ExitPriorityWrapper?

yeah. Not putting unused code in production as general good practice. Though I am not really strong on this. I will leave this to your decision.